.jenkins/Jenkinsfile

```diff
@@ -122,6 +128,11 @@ def DCAPBuildTest(String label, String build_type) {
     }
 }
 
+// Known issue: https://github.com/openenclave/openenclave/issues/2828
+// Add these back in when next OESDK release is available
```

Comment:
https://github.com/openenclave/openenclave/issues/2828 has been closed and the release is available as of 4/23

Reply:
Thanks! I've added these tests back in